Header header

# view angle can be between -1.0 and 1.0
# if value is 1 at viewAngleLeft, this means that the camera has to look to maximum
# angle LEFT. On the other side if value is -1 then the camera has to look right. 

float32 viewAngleLeft
float32 viewAngleUP

bool speedControlled
